deployer/templates/fluentd.yaml

```diff
@@ -63,6 +63,7 @@ objects:
             resources:
               limits:
                 cpu: 100m
+                memory: 512Mi
```

Comment:
For what its worth I deleted half of the fluentD pods and let them re-create with the new memory limit and have not seen them cross much above 100MB Ram utilization since yesterday. The other half are ranging from 300MB to 3GB but they have also been running for about 21days.